### DISCUSSION TITLE
Used Forma Bridge to bridge TIA, never received the TIA on Forma

### DISCUSSION THREAD

**Discussion by randomadrps** [blockscout/blockscout#10152] [Q&A]
(https://www.mintscan.io/celestia/tx/5F623D91AE6B00D12C793AB7CC50B0E65DBFB8C57DC9BA09B0CD6556144C7F0D?sector=logs)

Used Forma Bridge to bridge TIA, never received the TIA on Forma. My balance on Forma is still 0 TIA. Help pls thanks

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
